doc/content/devices/adding-devices/adding-devices-in-bulk/_index.md

```diff
@@ -87,3 +87,79 @@ See the following video from [The Things Network youtube channel](https://youtu.
 <details><summary>Show video</summary>
 {{< youtube "ouz-VuiosU4" >}}
 </details>
+
+## Claim devices using Claim Authentication Code
+
+Devices can be added in bulk together with their ownership information, through a process of device claiming using their claim authentication codes.
+
+Claiming devices can be done using {{% tts %}} Console or the CLI, for single or multiple devices.
+
+{{< tabs/container "Console" "CLI" >}}
+
+{{< tabs/tab "Console" >}}
+
+To claim a single device, first navigate to your application in {{% tts %}} Console and click the **+ Register end device** button.
+
+Select the **Enter end device specifics manually**, then provide **Frequency plan**, **LoRaWAN version** and **Regional Parameters version**.
+
+In the **Provisioning information** section, fill in the **JoinEUI** of The Things Join Server and **Confirm**. 
+
+Then, enter the **DevEUI**, **Claim authentication code** and **End device ID** of a device that you wish to claim.
```

Comment:
But what is the difference between this and the `## Onboarding devices using QR codes` in the `adding-devices`page?

Reply:
Oh I see. Sorry, you're right then.